scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/Worker.java

```diff
@@ -1,47 +1,56 @@
 package com.scylladb.cdc.model.worker;
 
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+import java.util.Collection;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
 import java.util.SortedSet;
+import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ScheduledExecutorService;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import com.google.common.base.Preconditions;
+import com.google.common.flogger.FluentLogger;
 import com.scylladb.cdc.model.GenerationId;
 import com.scylladb.cdc.model.StreamId;
 import com.scylladb.cdc.model.TableName;
 import com.scylladb.cdc.model.TaskId;
 import com.scylladb.cdc.model.Timestamp;
 
 public final class Worker {
+    private static final FluentLogger logger = FluentLogger.forEnclosingClass();
 
     private final WorkerConfiguration workerConfiguration;
+    private volatile boolean shouldStop = false;
 
     public Worker(WorkerConfiguration workerConfiguration) {
         this.workerConfiguration = Preconditions.checkNotNull(workerConfiguration);
     }
 
     /*
-     * Get a generation ID of given set of streams. It is assumed that all streams
-     * in a set belong to the same generation. The set is assumed to be non-empty.
+     * Get a generation ID of given set of streams. It is assumed that all
```

Comment:
Please avoid noise like this in the PRs.

Reply:
If you want to avoid such noise, I'd suggest adding in the formatting and cleanup options with the project, so the auto formatter does not foobar stuff by accident. I know - only ctrl-f selected lines, but given the newly discovered "save actions" thing, it seems to me the antithesis of eclipsy...